src/functions/indGraphFat.jl

```diff
@@ -3,40 +3,53 @@
 struct IndGraphFat{T <: RealOrComplex} <: IndGraph
   m::Int
   n::Int
-  A::AbstractArray{T,2}
-  AA::AbstractArray{T,2}
-  F::Base.LinAlg.Cholesky #LDL factorization
+  A::Array{T,2}
+  AA::Array{T,2}
+  tmp::Array{T,1}
+  # tmpx::Array{T,1}
+  F::Base.LinAlg.Cholesky{T, Array{T, 2}} #LL factorization
 end
 
-function IndGraphFat(A::AbstractArray{T,2}) where {T <: RealOrComplex}
+function IndGraphFat(A::Array{T,2}) where {T <: RealOrComplex}
   m, n = size(A)
   AA = A * A'
 
   F = LinAlg.cholfact(eye(m) + AA)
   #normrows = vec(sqrt.(sum(abs2.(A), 2)))
 
-  IndGraphFat(m, n, A, AA, F)
+  IndGraphFat(m, n, A, AA, Array{T, 1}(m), F)
 end
 
 # is_convex(f::IndGraph) = true
 # is_set(f::IndGraph) = true
 # is_cone(f::IndGraph) = false
 
 function prox!{T <: RealOrComplex}(
-    x::AbstractArray{T, 1},
-    y::AbstractArray{T, 1},
+    x::Array{T, 1},
+    y::Array{T, 1},
```

Comment:
There is no reason to restrict the input arguments to be the array type, this would inhibit views as input for example, the same goes for the other functions.